upstream/roles/produce_fbc/tasks/main.yml

```diff
@@ -65,6 +74,13 @@
     chdir: "{{ pfbc_extracted_fbc_location }}"
   register: pfbc_rc
 
+- name: Copy /database/index.db to a hidden IIB directory
```

Comment:
Name should be quoted here I think, at least for consistency.

Reply:
jaj I clicked on merge before noticed this comment. I'll fix it in a separate commit.